app/repository/meteors_repository.js

```diff
@@ -0,0 +1,21 @@
+import axios from "axios";
+import config from '../config/config.js';
+
+const fetchMeteors = async () => {
+    try {
+        const response = await axios.get(`${config.apiUrl}/feed`, {
+            params: {
+                start_date: config.startDate,
+                end_date: config.endDate,
+                api_key: config.apiKey,
+            }
+        });
+
+        return response.data.near_earth_objects;
+    } catch (error) {
+        throw new Error('Error fetching meteor data from repository');
+    }
```

Comment:
You can remove try..catch here and leave it only in the controller

Reply:
done